cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -626,6 +626,10 @@ body {
                     border-left-color: @green;
                 }
             }
+
+            h3.h5 {
```

Comment:
Is there a better way to target these elements?

Reply:
That's a good point! I updated the heading/footer class names in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/7f0fedc0e7419fcf8505398b8a4499afffce6a61